typescript/sdk/src/deploy/core/HyperlaneCoreDeployer.ts

```diff
@@ -179,28 +259,46 @@ export class HyperlaneCoreDeployer<
     const multisigIsm = await this.deployMultisigIsm(chain);
 
     const proxyAdmin = await this.deployContract(chain, 'proxyAdmin', []);
+    const deployerOwnedProxyAdmin = await this.deployContract(
+      chain,
+      'deployerOwnedProxyAdmin',
+      [],
+    );
```

Comment:
if we are not owner of proxyAdmin, lets deploy and pass through the deployer owned proxy admin